rmake.py

```diff
@@ -113,7 +116,10 @@ def config_cmd():
     tools = f"-DCMAKE_TOOLCHAIN_FILE={toolchain}"
     cmake_options.append( tools )
 
-    cmake_options.extend( cmake_platform_opts)
+    if os.name == 'nt':
```

Comment:
I think making cmake_platform_opts in line 114 to be a list like so:
`cmake_platform_opts = [f"-DROCM_DIR:PATH={rocm_path}", f"-DCPACK_PACKAGING_INSTALL_PREFIX={rocm_path}"]`
makes more sense then doing
`cmake_platform_opts.append(f"-DROCM_DIR:PATH={rocm_path} -DCPACK_PACKAGING_INSTALL_PREFIX={rocm_path}")`, since cmake_options would eventually have an element that holds two commands instead of one command per elemenet

Reply:
Good point - I hadn't noticed there were two options being added at the same time there. I agree that keeping them split makes sense.